roles/ansible-test-splitter/files/split_targets.py

```diff
@@ -3,21 +3,25 @@
 from pathlib import PosixPath
 import sys
 import json
+import argparse
+import os
 
-job_prefix = sys.argv[1]
-if len(sys.argv) == 3:
-    targets_from_cli = sys.argv[2].split(" ")
-else:
-    targets_from_cli = []
+parser = argparse.ArgumentParser()
```

Comment:
Thank you, this is a nice improvement.

Reply:
That was Aubin's commit, I agree it's good work!